qemu/tests/qemu_guest_agent.py

```diff
@@ -167,7 +168,7 @@ def _check_ga_service(self, session, cmd_check_status):
         s, o = session.cmd_status_output(cmd_check_status)
         return s == 0
 
-    def _get_main_qga_version(self, session, vm):
+    def _get_main_qga_version(self, session, vm, main_ver=True):
```

Comment:
I suppose better remove the word `main` from the function name or at least the Doc String of this function should be updated.

Reply:
okay, sure. changing it,